kasa/tests/discovery_fixtures.py

```diff
@@ -12,6 +12,8 @@
 from .fakeprotocol_smart import FakeSmartProtocol, FakeSmartTransport
 from .fixtureinfo import FixtureInfo, filter_fixtures, idgenerator
 
+DISCOVERY_MOCK_IP = "127.0.0.123"
```

Comment:
See my comment above.

Reply:
This was required as the `_post_update_hook` broke all the tests in `device_factory` due to the module requiring info not just in discovery info. I converted all those tests to use the `discovery` mock which fully mocks all the fixture queries.